### PR TITLE
Resolves #139: add method for getting vDC media

### LINKED ISSUE #139: Add vDC endpoint method for getting media

### PR DESCRIPTION
**Pull request by bsnyder788**

### CODE REVIEW

File: src/sdk/model/vdc/vdc.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ import { PerfCounter } from '../mixins/perf-samples/perf-counter';
 import { PerfSamplesRequest } from '../mixins/perf-samples/perf-samples-request';
 import { PerfSamplesSeries } from '../mixins/perf-samples/perf-samples-series';
 import { applyMixins } from 'rxjs/util/applyMixins';
+import { Media } from '../media';
+import { MediaJson } from '../media';
```
Comment (cfsnyder) [inline src/sdk/model/vdc/vdc.ts]: that import doesn't look right?

File: src/sdk/model/vdc/vdc.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ import { PerfCounter } from '../mixins/perf-samples/perf-counter';
 import { PerfSamplesRequest } from '../mixins/perf-samples/perf-samples-request';
 import { PerfSamplesSeries } from '../mixins/perf-samples/perf-samples-series';
 import { applyMixins } from 'rxjs/util/applyMixins';
+import { Media } from '../media';
+import { MediaJson } from '../media/__json__/media-json';
```
Comment (bsnyder788) [inline src/sdk/model/vdc/vdc.ts]: @cfsnyder updated

File: src/sdk/model/vdc/vdc.ts [hunk 3]
```diff
@@ -18,6 +18,8 @@ import { PerfCounter } from '../mixins/perf-samples/perf-counter';
 import { PerfSamplesRequest } from '../mixins/perf-samples/perf-samples-request';
 import { PerfSamplesSeries } from '../mixins/perf-samples/perf-samples-series';
 import { applyMixins } from 'rxjs/util/applyMixins';
+import { Media } from '../media';
```
Comment (cfsnyder) [inline src/sdk/model/vdc/vdc.ts]: this one is wrong too

File: src/sdk/model/vdc/vdc.ts [hunk 4]
```diff
@@ -18,6 +18,8 @@ import { PerfCounter } from '../mixins/perf-samples/perf-counter';
 import { PerfSamplesRequest } from '../mixins/perf-samples/perf-samples-request';
 import { PerfSamplesSeries } from '../mixins/perf-samples/perf-samples-series';
 import { applyMixins } from 'rxjs/util/applyMixins';
+import { Media } from '../media/media';
```
Comment (bsnyder788) [inline src/sdk/model/vdc/vdc.ts]: @cfsnyder fixed the path here too

**Pull request merged**